### ISSUE TITLE
Document Command-Line Arguments for `build_docker_image`.

### ISSUE THREAD

**Issue by Alexhuszagh** [cross-rs/cross#819]
Currently, the output for `cargo build-docker-image --help` is:

```
xtask-build-docker-image
Build a docker image from file

USAGE:
    xtask.exe build-docker-image [OPTIONS] [TARGETS]...

ARGS:
    <TARGETS>...    Targets to build for

OPTIONS:
        --dry-run
        --engine <ENGINE>            Container engine (such as docker or podman)
        --force
        --from-ci
    -h, --help                       Print help information
        --labels <LABELS>            Newline separated labels [env: LABELS=]
    -n, --no-output                  Set output to /dev/null
        --no-cache
        --no-fastfail
    -p, --push
        --progress <PROGRESS>        [default: auto] [possible values: auto, plain, tty]
        --repository <REPOSITORY>    [default: ghcr.io/cross-rs]
        --tag <TAG>                  Specify a tag to use instead of the derived one, eg `local`
    -v, --verbose                    Provide verbose diagnostic output
```

Most of these should have doc comments explaining what they mean, so others can know what they mean without reading source code.

**Comment by Emilgardis** [comment]
Are you working on this? I can do this otherwise

**Comment by Alexhuszagh** [comment]
> Are you working on this? I can do this otherwise

I am, should be done shortly.

**Comment by Alexhuszagh** [comment]
Closed in #820.
